Linux.md

```diff
@@ -96,3 +96,6 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="previousSuperuserCommand"></a> Runs Previous command used as superuser
```

Comment:
Improve heading to `Runs previous ran command`

Reply:
Done @arverma007